helper/server.go

```diff
@@ -0,0 +1,73 @@
+package helper
+
+import (
+	"os"
+	"runtime"
+
+	"github.com/gohugoio/hugo/commands"
+)
+
+//ResumeServer will start up the hugo server
+func ResumeServer(args []string) {
+	ToBuild()
+	CreateDirIfNotExist("content")
+	CreateDirIfNotExist("themes")
+	CreateDirIfNotExist("data")
+	CreateConfigFile()
+	if _, err := os.Stat("resume.json"); os.IsExist(err) {
+		if _, err := os.Stat("data/resume.json"); os.IsNotExist(err) {
+			err = os.Link("../resume.json", "data/resume.json")
```

Comment:
@stp-ip I thought after resumis init the file will always be created of the form resume.json thus this link.

Reply:
Yeah, but the assumption is that the tool will be run in the same directory as `resumic init` was run.
Another example could be git repository with old resume.json and the user's home directory to execute the `resumic serve` command.
Basically the tool shouldn't make assumptions around the file locations (besides a few sane defaults. Such as current directory, similar on how you did. Also I probably would copy the file instead of linking. Why the link?)